examples/tsx-clock/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
+      "declaration": false,
+      "jsx": "react",
+      "jsxFactory": "jsx",
+      "module": "amd",
+      "noImplicitAny": true,
+      "noUnusedLocals": true,
+      "outFile": "./build.js",
+      "preserveWatchOutput": true,
+      "removeComments": false,
+      "sourceMap": true,
+      "target": "ES6",
+      "lib": [
+          "dom",
+          "es2017",
+      ]
+  },
```

Comment:
Consider using my https://github.com/mightyiam/tsconfigs.

Reply:
That's a cool repo, but I kinda wanted the example to have the raw base tscconfig.json without any extends redirections for simplicity. Plus that package isn't in package.json right?